Update README.md (#12)
Snowflake-Labs/mlflow-snowflake: README.md

target_uri = 'snowflake'
deployment_client = get_deploy_client(target_uri)
```

#### SnowSQL Configuration file
[SnowSQL Configuration file](https://docs.snowflake.com/en/user-guide/snowsql-config.html) is a familiar concept among existing SnowSQL CLI users and a neccessary way to establish connection to Snowflake if you intend to use MLflow CLI for model deployment.
For the Snowflake deployment plugin, the `target_uri` needs to have the`snowflake` scheme.
Connection parameters can be specified by adding `?connection={CONNECTION_NAME}`.
The `CONNECTION_NAME` references the connection specified in the SnowSQL configuration file e.g. `snowflake:/?connection=connections.ml`.

#### Specification for target_uri
If you are using MLflow python API and already have created Snowflake session, then `target_uri` is `snowflake` which will ensure MLflow choose Snowflake as target platform.

> If you are using MLflow CLI, then you need to specify the connection name based on your SnowSQL configuration file. In this case, your `target_uri` would be `snowflake:/?connection={connection_name}`. The library would take care creating the Snowflake session for you.

Reviewer: can we create a simple util library to help creating the `target_uri` from a dict of connection parameters?

Author: `target_uri` is independent from connection parameters. it's most likely to be literal `snowflake` for majority of the customers.

Reviewer: For some reason, this paragraph is not making it clear.

Author: It's the same as above passage, try to rephrase more explicitly since got a question from one customer.

Reviewer 2: I think it would be better to extend the first passage then instead of near-duplicating it.

> 
> #### Specification for target_uri
> If you are using MLflow python API and already have created Snowflake session, then `target_uri` is `snowflake` which will ensure MLflow choose Snowflake as target platform.
> If you are using MLflow CLI, then you need to specify the connection name based on your SnowSQL configuration file. In this case, your `target_uri` would be `snowflake:/?connection={connection_name}`. The library would take care creating the Snowflake session for you.

Reviewer 2: ```suggestion

#### Specification for target_uri
If you are using the MLflow python API and already have created a Snowflake session, then `target_uri` is `snowflake`. 
If you are using the MLflow CLI, you need to specify the connection name based on your SnowSQL configuration file. In this case, your `target_uri` would be `snowflake:/?connection={connection_name}`. The library would take care of creating the Snowflake session for you.
```
